input/output-stride.incl

```diff
@@ -22,8 +22,7 @@ Output {
 Stopping {  cycle = 20; }
 Testing {   cycle_final = 20; }
 Testing {
-   time_final  = [0.0506399682687848, # OLD_PPM
-                  0.0507272430695593]; # NEW_PPM
+   time_final  = [0.0507272430695593];
```

Comment:
OLD_PPM referred to -- well -- old PPM methods that are no longer present in the code. This got removed in PR#24

Reply:
OLD_PPM referred to -- well -- old PPM methods that are no longer present in the code. This got removed in PR#24

Reply:
This comment referred to another comment above (in my interface hidden and shows only after "show" more) because here it seems that the results of NEW_PPM are the latest one.
At other places the "latest" does neither match NEW nor OLD.